usr/bin/nxdk-configure

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+. ${NXDK_DIR}/usr/share/nxdk/prelude.inl
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
+
+# Wrapper around autoconf ./configure scripts
```

Comment:
I got surprised when I ran this on zlib (?) `./configure`, because it turns out that it wasn't actually an autoconf project, or it was heavily modified.

This script should first verify if the `configure` file  is actually an autoconf script.